lti_consumer/tasks.py

```diff
@@ -9,10 +9,8 @@
 
 from lms import CELERY_APP
 
-ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)
 
-
-@CELERY_APP.task(name='lti_consumer.tasks.send_email_message', routing_key=ACE_ROUTING_KEY)
```

Comment:
Please clarify why is routing_key removed?

Reply:
Because we had the next error with this parameter when the celery worker was starting:

```
ERROR:stevedore.extension:Could not load 'rg_learning_time': The SECRET_KEY setting must not be empty.
ERROR:stevedore.extension:The SECRET_KEY setting must not be empty.
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/stevedore/extension.py", line 163, in _load_plugins
    verify_requirements,
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/stevedore/extension.py", line 184, in _load_one_plugin
    plugin = ep.resolve()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/edx/app/edxapp/edx-platform/lms/lib/lti-consumer/lti_consumer/__init__.py", line 4, in <module>
    from .lti_consumer import LtiConsumerXBlock
  File "/edx/app/edxapp/edx-platform/lms/lib/lti-consumer/lti_consumer/lti_consumer.py", line 75, in <module>
    from .outcomes import OutcomeService
  File "/edx/app/edxapp/edx-platform/lms/lib/lti-consumer/lti_consumer/outcomes.py", line 20, in <module>
    from .tasks import send_email_message
  File "/edx/app/edxapp/edx-platform/lms/lib/lti-consumer/lti_consumer/tasks.py", line 11, in <module>
    ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 129, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
ImproperlyConfigured: The SECRET_KEY setting must not be empty.

```

Reply:
It seems that these error logs don't relate to the LTI consumer, it's an issue from the building packages order at the same time ACE_ROUTING_KEY looks like important for the ACE service, doesn't it?

Reply:
@idegtiarov 
I've made the mistake. This first line doesn't relate to this problem, you're right. I mean this one:
```
ERROR:stevedore.extension:Could not load 'rg_learning_time': The SECRET_KEY setting must not be empty.

```

But this line from stack trace related to `ImproperlyConfigured: The SECRET_KEY setting must not be empty.` error:
```
File "/edx/app/edxapp/edx-platform/lms/lib/lti-consumer/lti_consumer/tasks.py", line 11, in <module>
    ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)
```

That's why ACE_ROUTING_KEY was deleted to resolve this task quickly.

Reply:
Deleting parameter is not an option6 it seems that import order could be a root cause of the issue. Could you please clarify this.

Reply:
I don't understand. What do you mean? How can we change the import order of "from django.conf import settings " or/and "ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)" ? And how It can relate to this error?